doc/Android.md

```diff
@@ -0,0 +1,109 @@
+**Document based on Minetest `5.6.1` Android build**
+# Minetest: Android version
+Minetest for Android are the same Minetest on Windows but compiled for Android, not seperate version
+## Controls
+The Android port doesn't support everything you can do on PC due to the
+limited capabilities of common devices. What can be done is described
+below:
+
+While you're playing the game normally (that is, no menu or inventory is
+shown), the following controls are available:
+* Look around: touch screen and slide finger
+* double tap: place a node
+* long tap: dig node or use the holding item
+* touch buttons: press button
+* Buttons:
+
+1. left upper corner: chat
+2. right lower corner: jump
+3. right lower corner: crouch
+4. left lower corner (Joystick): walk/step...
+5. left lower corner: display inventory
+
+When a menu or inventory is displayed:
+* double tap outside menu area: close menu
+* tap on an item stack: select that stack
+* tap on an empty slot: if you selected a stack already, that stack is placed here
+* drag and drop: touch stack and hold finger down, move the stack to another
+  slot, tap another finger while keeping first finger on screen
+  --> places a single item from dragged stack into current (first touched) slot. If a stack is selected, the stack will be split as half and one of the splitted stack will be selected
+### Limitaions
+* Android player have to double tap to place node, this can be annoying in some game/mod
+* Some old Android device only support 2 touch at a time, some game/mod contain button combination that need 3 touch (example: jump + Aux1 + hold)
+* Complicated control like pick up an cart in MTG can be difficult or impossible on Android device
+## File Path
+There are some settings especially useful for Android users. Minetest's config
+file can usually be found at:
+
+* old version (Before 5.4.2): ``/sdcard/Minetest/`` or ``/storage/emulated/0/`` if stored in device, if stored in SD card: ``/storage/emulated/(varying folder name)/``
+* new version (After 5.4.2): ``/sdcard/Android/data/net.minetest.minetest/`` or ``/storage/emulated/0/Android/data/net.minetest.minetest/`` if stored in device, if stored in SD card: ``/storage/emulated/(varying folder name)/Android/data/net.minetest.minetest/``
```

Comment:
1. "new" is quickly outdated, hence prefer to write "Before X.Y.Z" and "After X.Y.Z"
2. Use separate listing points for each path to be reader-friendlier
3. Insert newlines before new headings to make the plaintext version easier to read